backend/tests/unit/services/test_dashboard_update_service.py

```
@@ -273,22 +147,21 @@ async def test_process_llm_interaction_success(self, update_service, mock_broadc
             'error_message': None
         }
         
-        # Test batched processing (default)
+        # Test immediate processing (no batching)
         sent_count = await update_service.process_llm_interaction(session_id, interaction_data)
```

Ruff (0.12.2)

Line too long (95 > 88)

(E501)